queue_services/entity-emailer/src/entity_emailer/email_templates/CONT-IN-COMPLETED.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <base href="/">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="referrer" content="origin-when-cross-origin"/>
+    <meta name="author" content="BC Registries and Online Services">
+    <title>Continuation Documents from the Business Registry</title>
+    [[style.html]]
+  </head>
+
+  <body>
+    <table class="body-table" role="presentation">
+      <tr>
+        <td>
+          [[header.html]]
+
+          <div class="container">
+            <p class="title-message bold">
+              <span>Continuation Documents from the Business Registry<span>
```

Comment:
As per spreadsheet this should be `You have successfully registered your business with BC Business Registry.`